pinot-spi/src/main/java/org/apache/pinot/spi/data/Schema.java

```diff
@@ -735,7 +735,8 @@ public static DateTimeFieldSpec convertToDateTimeFieldSpec(TimeFieldSpec timeFie
     String outgoingTimeFormat = outgoingGranularitySpec.getTimeFormat();
     String[] split = StringUtils.split(outgoingTimeFormat, DateTimeFormatSpec.COLON_SEPARATOR, 2);
     DateTimeFormatSpec formatSpec;
-    if (split[0].equals(DateTimeFieldSpec.TimeFormat.EPOCH.toString())) {
+    if (split[0].equals(DateTimeFieldSpec.TimeFormat.EPOCH.toString()) || split[0].equals(
```

Comment:
This is for the deprecated `TimeFieldSpec` which does not have the `TIMESTAMP` support. No need to change this method